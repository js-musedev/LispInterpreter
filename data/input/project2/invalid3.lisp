EREF EDFJDKFJ 54564 [][= `~!bgg4tyy6354+64`