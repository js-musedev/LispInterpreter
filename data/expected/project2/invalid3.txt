Expected either an ATOM or OPEN token.
Actual: ERROR_TOKEN    Value: [][=